userspace/libscap/examples/01-open/README.md

```diff
@@ -111,6 +111,8 @@ Number of dropped events caused by full buffer (n_drops_buffer_dir_file_enter sy
 Number of dropped events caused by full buffer (n_drops_buffer_dir_file_exit syscall category): 0
 Number of dropped events caused by full buffer (n_drops_buffer_other_interest_enter syscall category): 0
 Number of dropped events caused by full buffer (n_drops_buffer_other_interest_exit syscall category): 0
+Number of dropped events caused by full buffer (n_drops_buffer_close_exit syscall category): 0
```

Comment:
oh @Andreagit97 we need to update the README as we now have the new scap stats v2 output in the scap-open example binary. That's on my I forgot it back then ...

Reply:
done :)